docs/docs/includes/introduction-self-guided.md

```diff
@@ -93,11 +93,19 @@ We have provided a bash script to automate the deployment of the resources requi
 
 === "Automated deployment"
 
-    The script `deploy.sh` deploys to the `eastus2` region by default; edit the file to change the region or resource names. To run the script, open the VS Code terminal and run the following command:
+    By default, the deployment script uses the `eastus2` region. You can edit `deploy.sh` or `deploy.ps1` to change the region or resource names.
 
-    ```bash
-    cd infra && ./deploy.sh
-    ```
+    - To deploy using **Bash** (Linux/macOS or GitHub Codespaces), run the following command in the VS Code terminal:
+
+        ```bash
+        cd infra && ./deploy.sh
+        ```
+
+    - To deploy using **PowerShell** (Windows), run:
+
+        ```powershell
+        Set-Location infra; ./deploy.ps1
+        ```
 
```

Comment:
We could do this using tabs:

```suggestion
    === "Linux/MacOS"

    By default, the deployment script uses the `eastus2` region. You can edit `deploy.sh` to change the region or resource names.

    Run the following command in the VS Code terminal:

        ```bash
        cd infra && ./deploy.sh
        ```

    === "Windows"

    By default, the deployment script uses the `eastus2` region. You can edit `deploy.ps1` to change the region or resource names.

    Run the following command in a terminal:

        ```powershell
        Set-Location infra; ./deploy.ps1
        ```

```

Reply:
Yeah but Powershell Core also runs on Linux too 🙂 The content of deploy.ps1 is that it can be run on Linux. The file separators use back slash. But I think for now we should go for this since #SeasonofAgents and #MSAISkillFest is on.